src/helpers/strings-pt-pt.ts

```diff
@@ -522,7 +522,13 @@ let strings: { [key: string]: string } = {
     "device-activity": "Atividade do dispositivo",
     "daily": "Diário",
     "weekly": "Semanal",
-    "monthly": "Mensal"
+    "monthly": "Mensal",
+    "bonus": "Bónus",
+    "syncing-data": "Sincronização de dados...",
+    "health-connect-phr-sync-title": "Sincronizar com o Health Connect",
+    "health-connect-phr-sync-prompt": "Escolha os registos de saúde para ler e escrever no Health Connect",
+    "setup": "Configurar",
+    "air-quality": "Qualidade do Ar"
```

Comment:
Though it isn't required for these to work, I find it helpful if the language strings are in the same position in all the files.  Any chance we could insert these missing strings at the same locations where they exist in the English file?

Reply:
The  inconsistent order bugs me too, but they're not all in the same places now regardless. 
Suggest a follow-up PR to put all the string files in alphabetical order, and then hopefully it'll be easier to maintain going forward?

Reply:
> Suggest a follow-up PR to put all the string files in alphabetical order, and then hopefully it'll be easier to maintain going forward?

I like that idea.  Alpha sort may "ungroup" some keys that were purposefully co-located when added, but at least the order would then be consistent across the files, so I'd be fine with that.